tests/vmi_configuration_test.go

```diff
@@ -1923,7 +1922,7 @@ var _ = Describe("[sig-compute]Configurations", func() {
 			Expect(err).ToNot(HaveOccurred())
 
 			Eventually(ThisDV(dataVolume), 240).Should(Or(HaveSucceeded(), BeInPhase(cdiv1.WaitForFirstConsumer)))
-		}, 60)
```

Comment:
I've seen this in a couple other places now that those default timeout values have been removed. Why do we remove the explicit timeout values? I guess there was a reason to define them as they were?

Also they differ widely, I've seen some as small as `15` while the biggest was `720`? So what is the default timeout value now?

I.e. https://github.com/kubevirt/kubevirt/pull/7326/files#diff-d9916fc2bf48043a4899c599755af277e7b8c341d8a737999f7a6667f61596f1L110

Reply:
The timeout decorator is deprecated and not supported anymore,
as well as the async testing feature (with the done channel)
https://onsi.github.io/ginkgo/MIGRATING_TO_V2#removed-async-testing

some of the timeout values might be a legacy copy paste,
felt it is better to ad hoc update tests in case needed instead of updating all the tests with limitation just because there used to be one, (which didn't cause timeouts before as well),

I don't think that  know if there is a per test default timeout now ?
see please https://github.com/golang/go/issues/48157

couldn't open the link you pasted, what is the file / line please ?

Reply:
I don't see any mention that timeout is deprecated, I only see this note here: https://onsi.github.io/ginkgo/MIGRATING_TO_V2#timeout-behavior

There is no statement I can find that the previous timeout behavior without the `done` channel does not exist any more.

Reply:
Having the timeout gives a runtime error, about unknown decorator

Reply:
Having the timeout gives a runtime error, about unknown decorator (`BeforeEach` has new supported decorators)
```
Unknown Decorator
tests/vmi_configuration_test.go:1914
  [BeforeEach] node was passed an unknown decorator: '60'

  Learn more at: http://onsi.github.io/ginkgo/#node-decorators-overview

```

also see here please
https://onsi.github.io/ginkgo/MIGRATING_TO_V2#removed-async-testing
the timeout won't be enforced